src/components/common/NameRequestInfo.vue

```diff
@@ -80,42 +82,29 @@
         </v-row>
       </div>
     </template>
-
-    <!-- Name Translation -->
-    <NameTranslation
-      v-if="showNameTranslation"
-      @hasNameTranslation="emitHasNameTranslation($event)"
-    />
```

Comment:
This was moved to the parent component so that THIS component could be NR stuff only.